04.ls/ls.rb

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+NUMBER_OF_ROW = 3
+target_path = ARGV[0] ||= './'
```

Comment:
```suggestion
target_path = ARGV[0] || './'
```

じゃないですかね？

あと変数の寿命を短くしましょう。
https://bootcamp.fjord.jp/pages/311

Reply:
nilガードと||による短絡評価を混同していて間違ってしましました。
修正しました。